packages/mdc-tab/README.md

```diff
@@ -104,6 +105,7 @@ Mixin | Description
 `mdc-tab-text-label-color($color)` | Customizes the color of the tab text label.
 `mdc-tab-icon-color($color)` | Customizes the color of the tab icon.
 `mdc-tab-parent-positioning` | Sets the positioning of the MDCTab's parent element so that `MDCTab.computeDimensions()` reports the same values in all browsers.
+`mdc-tab-fixed-width($width)` | Sets the fixed width of the tab. The tab will never be smaller than the given width, although it may be larger if the content of the tab is larger than the given width.
```

Comment:
Should this be named `mdc-tab-min-width` instead?

Reply:
It does set the tab to a fixed width from which it won't grow or shrink. I think min-width isn't quite as accurate.